README.md

```diff
@@ -344,10 +568,11 @@ nv-ipam accepts the following CNI configuration:
 ```
 
 * `type` (string, required): CNI plugin name, MUST be `"nv-ipam"`
-* `poolName` (string, optional): name of the IP Pool to be used for IP allocation.
+* `poolName` (string, optional): name of the Pool to be used for IP allocation.
 It is possible to allocate two IPs for the interface from different pools by specifying pool names separated by coma,
 e.g. `"my-ipv4-pool,my-ipv6-pool"`. The primary intent to support multiple pools is a dual-stack use-case when an 
 interface should have two IP addresses: one IPv4 and one IPv6. (default: network name as provided in CNI call)
+* `poolType` (string, optional): type (IPPool, CIDRPool) of the pool which is referred by the `poolName`. The field is case-insensitive. (default: `"IPPool"`)
```

Comment:
Same comment as above, I think values should be lowercase.

Reply:
done